DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/EditDelegate/Index.cshtml

```diff
@@ -57,6 +59,8 @@
                        autocomplete=""
                        hint-text=""
                        css-class="nhsuk-u-width-one-half" />
+
+        <partial name="_EditRegistrationNumber" model="@Model" />

```

Comment:
Just noting that it's slightly weird that the label and hint for the prn input address the user as "you", when the user is actually editing someone else's details. Not sure if it's worth modifying the partial to change the wording?

Reply:
![image](https://user-images.githubusercontent.com/59561751/155522856-3b342801-3108-439a-9fc4-81013fde2505.png)